_posts/2017-09-29-thank-you-theano.md

```diff
@@ -0,0 +1,7 @@
+# Theano's contribution
+
+The Chainer team is saddened to hear about the end of Theano development. Some of us used Theano when we first started studying deep learning and many aspects of Chainer were inspired by Theano’s clean interface design.
+
+Like Theano, Chainer is a Python neural network framework with a numpy-like syntax for computation on CPUs or GPUs. Thanks to our previous work with Theano, Chainer has a TheanoFunction wrapper which enables Theano functions to be used within Chainer as is. We have started writing a “Chainer for Theano users” articletutorial in case Theano users are interested in trying Chainer.
```

Comment:
articletutorial?

Reply:
a TheanoFunction wrapper -> TheanoFunction

Reply:
Change last sentence to:

In case Theano users are interested in trying Chainer, the Chainer documentation follows a similar style to Theano.

Chainer documentation should be hyperlinked to docs.chainer.org site.

Tweet text:

Theano inspired Chainer development in many ways. Thanks to the team for being one of the first neural network frameworks: https://short.url/